extensions/timeseries/src/test/java/ai/djl/timeseries/block/BlockTest.java

```diff
@@ -20,17 +20,24 @@
 import ai.djl.ndarray.types.Shape;
 import ai.djl.nn.Parameter;
 import ai.djl.testing.Assertions;
+import ai.djl.testing.TestRequirements;
 import ai.djl.training.ParameterStore;
 import ai.djl.training.initializer.Initializer;
 
 import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 
 import java.util.Arrays;
 import java.util.List;
 
 public class BlockTest {
 
+    @BeforeClass
+    public void setUp() {
+        TestRequirements.notArm();
```

Comment:
Add TODO here, we should be able to support ARM when we add PyTorch support

Reply:
Good point - I'll add those to all the timeseries tests